usaspending_api/accounts/views/federal_accounts_v2.py

```diff
@@ -78,7 +85,27 @@ def get(self, request, pk, format=None):
 class FiscalYearSnapshotFederalAccountsViewSet(APIView):
     @cache_response()
     def get(self, request, pk, format=None):
-        return None
+
+        # import pytest; pytest.set_trace()
+        queryset = AppropriationAccountBalances.objects.filter(treasury_account_identifier__federal_account_id=int(
+            pk)).filter(final_of_fy=True).filter(submission__reporting_fiscal_year=fy(datetime.today()))
+        queryset = queryset.aggregate(
+            outlay=Sum('gross_outlay_amount_by_tas_cpe'),
+            budget_authority=Sum('budget_authority_available_amount_total_cpe'),
+            obligated=Sum('obligations_incurred_total_by_tas_cpe'),
+            unobligated=Sum('unobligated_balance_cpe'),
+            balance_brought_forward=Sum(
+                    F('budget_authority_unobligated_balance_brought_forward_fyb')
+                    +
+                    F('adjustments_to_unobligated_balance_brought_forward_cpe')
+                    ),
+            other_budgetary_resources=Sum('other_budgetary_resources_amount_cpe'),
+            appropriations=Sum('budget_authority_appropriated_amount_cpe')
+            )
+        if queryset['outlay'] is not None:
```

Comment:
Since the above statement uses `aggregate` on the queryset the returned value is no longer a django queryset. Could this dict variable be renamed?